src/Codeception/Test/Descriptor.php

```diff
@@ -32,15 +32,22 @@ public static function getTestSignature(\PHPUnit\Framework\SelfDescribing $testC
      */
     public static function getTestSignatureUnique(\PHPUnit\Framework\SelfDescribing $testCase)
     {
+        $env     = null;
         $example = null;
 
+        if (method_exists($testCase, 'getScenario')
+            && $env = $testCase->getScenario()->current('env')
```

Comment:
If `$env` evaluates to some falsy value, it skips concatenation below, but is used in return line.
It would be better to assign temporary value to a different variable, e.g. `$currentEnv`.